rtv/subreddit.py

```diff
@@ -135,5 +135,8 @@ def draw_item(win, data, inverted=False):
         if row in valid_rows:
             text = '{author}'.format(**data)
             win.addnstr(row, 1, text, n_cols-1, curses.A_BOLD)
+            if data['flair'] != None:
```

Comment:
Thanks for the feedback.  I'm going to go through and fix these points
tomorrow.

On Wed, Mar 11, 2015 at 6:46 PM, michael-lazar <notifications@github.com>
wrote:

> In rtv/subreddit.py
> <https://github.com/michael-lazar/rtv/pull/34#discussion_r26263968>:
>
> > @@ -135,5 +135,8 @@ def draw_item(win, data, inverted=False):
> >          if row in valid_rows:
> >              text = '{author}'.format(**data)
> >              win.addnstr(row, 1, text, n_cols-1, curses.A_BOLD)
> > +            if data['flair'] != None:
>
> data['flair'] should never be None, correct?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/michael-lazar/rtv/pull/34/files#r26263968>.
>